browser-test/src/support/admin_programs.ts

```diff
@@ -160,11 +160,17 @@ export class AdminPrograms {
 
   // TODO(clouser): More fine-grained selectors for this and active.
```

Comment:
While here, can you remove the TODO against my name? I think I forgot to remove it prior to originally merging the PR.

Reply:
done